### PR TITLE
AWS Migration: Final Task Checklist [ch1890]

### PR DESCRIPTION
**Pull request by joesb**
This PR is just a 'work in progress' item for now, for visibility

### CODE REVIEW

File: web/sites/default/settings.prod-eks.php [hunk 1]
```diff
@@ -35,8 +34,10 @@
 $config['config_split.config_split.crm_prod']['status'] = TRUE;
 $config['config_split.config_split.crm_stage']['status'] = FALSE;
 
+if (getenv('CONNECT_DOMAIN')) {
```
Comment (stutily) [inline web/sites/default/settings.prod-eks.php]: I think we probably want to set this before going live?

Comment (joesb) [reply web/sites/default/settings.prod-eks.php]: Yep, on PaaS that looks like

```
CONNECT_DOMAIN: connect.nationalleadership.gov.uk
```

**Comment by joesb** [comment]
I think it makes sense to merge current master into this branch at some point soon, if you're able @stutily

**Comment by stutily** [comment]
^^ merged master in - ready to merge this back to master (and staging and production) which should fix the build errors of this morning .

**Comment by joesb** [comment]
Ok, great. Will merge now, and then add a PR to `production`

**Pull request merged**